server/controllers/auth.js

```diff
@@ -36,28 +36,23 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
 
   const isPetSitter = req.query.accountType === "petSitter" ? true : false;
 
-
   if (user) {
+    const customer = await stripe.customers.create({
+      description: `Customer name is ${name}`,
+    });
+    const { id } = customer;
     if (isPetSitter) {
       await PetSitter.create({
         userId: user._id,
+        stripeConnectId: id,
         name,
       });
```

Comment:
This previouslly had a syntax error. Stripe should now create a users upon registration, same with PetSitter.